### PR TITLE
Add scale from zero to Lab 9

### LINKED ISSUE #62: Add scale from zero to Lab 9
This is a good feature to cover in the labs.

This could just be as simple as scaling manually using `docker service scale` or `kubectl scale` and then invoking the function used in the examples (nodeinfo).

Stretch goal - mention faas-idler and how to turn "dry run" to false.

### PR DESCRIPTION
**Pull request by ivanayov**
This adds instructions how to test scale from 0 and how to use
faas-idler as well

Signed-off-by: Ivana Yovcheva <iyovcheva@vmware.com>

Closes #62

### CODE REVIEW

File: lab9.md [hunk 1]
```diff
@@ -64,4 +64,60 @@ Now you can verify using `$ docker service ps nodeinfo` that new replicas of `no
...
+Try it out with the following commands:
+
+Docker Swarm:
+```
+$ docker service scale nodeinfo=0
+```
+
+Kubernetes:
+```
+$ POD=$( kubectl get pods  -n openfaas-fn -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep astronaut-finder)
```
Comment (alexellis) [inline lab9.md]: Please scale the deployment, not the pod. It should be a single line

File: lab9.md [hunk 2]
```diff
@@ -64,4 +64,60 @@ Now you can verify using `$ docker service ps nodeinfo` that new replicas of `no
...
+$ kubectl scale --replicas=0 $POD -n openfaas-fn
+```
+
+Open the OpenFaaS UI and check that nodeinfo has 0 replicas, or by `docker service ls | grep nodeinfo`.
+
+> For Kubernetes use `kubectl get pods -n openfaas-fn | grep nodeinfo`
```
Comment (alexellis) [inline lab9.md]: Just use `kubectl get deploy`

File: lab9.md [hunk 3]
```diff
@@ -64,4 +64,60 @@ Now you can verify using `$ docker service ps nodeinfo` that new replicas of `no
...
+
+Now invoke the function and check back that it scaled to 1 replicas.
+
+### Setup faas-idler
```
Comment (alexellis) [inline lab9.md]: I think this is OK, but we should make it an optional / advanced lab towards the end of the material. @rgee0 would you agree?

Comment (alexellis) [reply lab9.md]: I'm tempted to ask for this in a new Lab 12, and we should check but I think the idler is now part of the stack for both Swarm + K8s.

**Comment by ivanayov** [comment]
@alexellis I'm not sure what you mean by the format. Do you mean this type of subsections: `#### _Setup faas-idler_`?

**Comment by alexellis** [comment]
I thought you had used `###` type of headers to separate the K8s/Swarm instructions in the newer PR? (I could be wrong) 😄

**Comment by ivanayov** [comment]
It's too nested on places and that's why just `###` does not work there and I've used `#### _` everywhere for consistency.

File: lab12.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Lab 12 - Advanced feature - Scale idle functions to 0
```
Comment (alexellis) [inline lab12.md]: I'm going to suggest removing Lab 12 completely. I don't think it's adding value over what is in the official documentation.

File: lab9.md [hunk 5]
```diff
@@ -64,4 +64,26 @@ Now you can verify using `$ docker service ps figlet` that new replicas of `figl
 
 Now stop the bash script and you will see the replica count return to 1 replica after a few seconds.
 
+### Try scale from zero
```
Comment (alexellis) [inline lab9.md]: Much more concise 👍

**Comment by alexellis** [comment]
Merging. @rgee0 I think everything was covered, but please feel free to comment if not.

**Pull request merged**